assets/js/components/HealthSummary/HomeHealthSummary.jsx

```diff
@@ -1,9 +1,11 @@
-import React, { Fragment } from 'react';
+import React, { Fragment, useState, useEffect } from 'react';
 import Table from '@components/Table';
 import HealthIcon from '../Health/HealthIcon';
 import { Link } from 'react-router-dom';
 import { useSelector } from 'react-redux';
 import HealthSummary from '@components/HealthSummary';
+import useQueryStringValues from '../../hooks/useQueryStringValues';
```

Comment:
I just realized we should probably use `@hooks` in the path for the import here.